prime-router/src/test/kotlin/fhirengine/azure/FHIRTranslatorIntegrationTests.kt

```diff
@@ -798,4 +798,166 @@ class FHIRTranslatorIntegrationTests : Logging {
             assertThat(translatedValue).isEqualTo(reportContents.toByteArray())
         }
     }
+
+    @Test
+    fun `successfully translate HL7 for FHIR receiver when isSendOriginal is true from convert step`() {
```

Comment:
These tests ensure `sendOriginal` items still function when `convert` is the root report, as the case would be when the submissions API is used.